benchmarks/utils.py

```diff
@@ -0,0 +1,9 @@
+import os
+import django
+
+def bench_setup():
```

Comment:
Nice. 

Maybe add this as a separate PR first so we can migrate all of the existing files to do this?

Reply:
I have added a separate PR for this